### PR TITLE
include parameters in execution status header

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Several folks have asked about this. Shows the parameters that were used when a pipeline ran.

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
@zanthrash please review

File: app/scripts/modules/core/delivery/status/executionStatus.controller.js [hunk 1]
```diff
@@ -50,4 +50,10 @@ module.exports = angular.module('spinnaker.core.delivery.executionStatus.control
       return 'Unknown';
     };
 
+    if ($scope.execution.trigger && $scope.execution.trigger.parameters) {
+      $scope.parameters = Object.keys($scope.execution.trigger.parameters).sort().map((paramKey) => {
+        return { key: paramKey, value: $scope.execution.trigger.parameters[paramKey] };
```
Comment (zanthrash) [inline app/scripts/modules/core/delivery/status/executionStatus.controller.js]: Is there a reason that 'parameters' is being added to the $scope and not to the 'controllerAs' name?

Comment (anotherchrisberry) [inline app/scripts/modules/core/delivery/status/executionStatus.controller.js]: ugh, fine, I'll refactor the whole thing.

Comment (zanthrash) [inline app/scripts/modules/core/delivery/status/executionStatus.controller.js]: If we ever want to move to Angluar2 then we are going to have to do all this stuff anyway.

**Comment by anotherchrisberry** [comment]
Turns out almost nothing in this controller was actually being used.

**Comment by anotherchrisberry** [comment]
Updated and ready for re-review

**Comment by anotherchrisberry** [comment]
@zanthrash ping

**Comment by zanthrash** [comment]
LGTM. Just needs a rebase.

**Pull request merged**